apps/op-app/src/adapters/express/routes/interaction.ts

```diff
@@ -169,10 +174,21 @@ export default function createInteractionRouter(
         new HttpBadRequestError(`Unable to parse the "_io_fims_token" cookie.`),
       );
       req.log.debug("_io_fims_token parsed from cookies");
+      const rpParams = requestParamsSchema.safeParse(interaction.params);
+      assert.ok(
+        rpParams.success,
+        new HttpBadRequestError(`Unable to parse the query params.`),
+      );
+      const ipAddress = req.ip || "";
```

Comment:
we should raise an error if `ipAddress` is empty (because `""` is not valid/accettable value for an `ipAddress`)